stdlib/public/stubs/CommandLine.cpp

```diff
@@ -67,10 +72,12 @@ extern "C" char ** _swift_stdlib_getUnsafeArgvArgc(int *outArgLen) {
     return _swift_stdlib_ProcessOverrideUnsafeArgv;
   }
 
-  FILE *cmdline = fopen("/proc/self/cmdline", "rb");
+  FILE *cmdline = fopen(PROCFS_CMDLINE_PATH, "rb");
   if (!cmdline) {
     swift::fatalError(0,
-            "fatal error: Unable to open interface to '/proc/self/cmdline'.\n");
+            "fatal error: Unable to open interface to '"
+            PROCFS_CMDLINE_PATH
+            "'.\n");
```

Comment:
Changes to this file LGTM, feel free to submit as a separate PR if you would like them to be merged immediately.